cmd/panda_executor/panda_executor.go

```diff
@@ -0,0 +1,184 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	controller "github.com/panda-re/panda_studio/internal/panda_controller"
+)
+
+const QCOW_NAME = "bionic-server-cloudimg-amd64-noaslr-nokaslr.qcow2"
+const RECORDING_NAME = "test"
+const EXECUTION_LOG = "execution.log"
+
+var QCOW_LOCAL = fmt.Sprintf("/root/.panda/%s", QCOW_NAME)
+
+func main() {
+	ctx := context.Background()
+
+	runRecording(ctx)
+
+	runReplay(ctx)
+}
+
```

Comment:
Do we have somewhere that explains the process of runRecording? As in a text file saying that it starts the agent, connects, etc? If not, is that something we would want to add?

Reply:
There is not a markdown for this file or anything like that. There is a brief mention of its purpose in the root README and some comments could be added to this file